lib/jekyll-heroicons.rb

```
@@ -63,13 +63,9 @@ def prepare(markup)
 
     # This method checks if heroicons liquid tag has a variant specified. If not, it check default variant in the config file. If both of those are not defined, it falls back to the default variant.
     def variant(markup)
-      if @options.key?(:variant)
-        @options[:variant]
-      elsif config["variant"]
-        config["variant"]
-      else
-        DEFAULT_VARIANT
-      end
+      @options[:variant] || config["variant"] || DEFAULT_VARIANT
```

**suggestion:** we're swallowing error here - it doesn't seem nice ;-)

How about an alternative approach, e.g.:
```
@options[:variant] || config&.dig("variant") || DEFAULT_VARIANT
```

so we can remove `rescue`, the error swallowing part?

---

I agree! That approach looks cleaner and avoids swallowing the error. I've already updated it. However, removing rescue causes the same issue in my environment. It seems like an environment-specific problem, so I'll investigate further!

---

@hahwul 
this would resolve it, notice a `&.` it will actually not raise an error if config is nil.

`@options&.dig(:variant) || config&.dig("variant") || DEFAULT_VARIANT`

---

Oh, I apologize for that. It works well with the code you modified. I made a mistake because I was only thinking about simply deleting the rescue.